resources/function_help/json/is_valid

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "is_valid",
+  "type": "function",
+  "description": "Returns true if a geometry is valid. Uses GEOS.",
```

Comment:
I'd drop the "uses geos" part -- it won't mean anything for most users. Maybe you could work in postgis' description of this function: "Tests if a geometry value is well-formed in 2D according to the OGC rules for geometry validity."